docs/standard/garbage-collection/unmanaged.md

```diff
@@ -31,15 +31,15 @@ If your types use unmanaged resources, you should do the following:
   - Define a [finalizer](../../csharp/programming-guide/classes-and-structs/finalizers.md). Finalization enables the non-deterministic release of unmanaged resources when the consumer of a type fails to call <xref:System.IDisposable.Dispose%2A?displayProperty=nameWithType> to dispose of them deterministically.
 
     > [!WARNING]
-    > However, because object finalization can be a complex and an error-prone operation, we recommend that you use a safe handle instead of providing your own finalizer.
+    > Object finalization can be a complex and error-prone operation, we recommend that you use a safe handle instead of providing your own finalizer.
```

Comment:
```suggestion
    > Object finalization can be a complex and error-prone operation. We recommend that you use a safe handle instead of providing your own finalizer.
```